### PR TITLE
Add caution to Clear message

### PR DESCRIPTION
**Pull request by DennieDan**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -361,7 +361,15 @@ to find your schedule for a specific day of the week.
...
 ### Clearing all entries : `clear`
 
-Clears all entries from the address book.
+Clears all data from the data file of UGTeach.
+
+<box type="warning" header ="##### Caution">
+This action is not undoable and your deleted data cannot be recovered. You should save a copy of the data file before trying this command.
+
+* Locate to the folder to save `ugteach.jar` and go to the folder `data`.
+* Inside the folder, duplicate the data file called `ugteach.json` and change the file name.
```
Comment (Soumyadip-cmd) [inline docs/UserGuide.md:370]: The part about saving a copy may not be needed. Just need to warn that action is irreversible and data cannot be recovered.

**Review by Soumyadip-cmd — COMMENTED** [review]
Some considerations

File: docs/UserGuide.md [hunk 2]
```diff
@@ -361,7 +361,15 @@ to find your schedule for a specific day of the week.
 
 ### Clearing all entries : `clear`
 
-Clears all entries from the address book.
+Clears all data from the data file of UGTeach.
+
+<box type="warning" header ="##### Caution">
+This action is not undoable and your deleted data cannot be recovered. You should save a copy of the data file before trying this command.
```
Comment (teamotitan) [inline docs/UserGuide.md:374]: Do you want to try

"You cannot undo this action and your...."

File: docs/UserGuide.md [hunk 3]
```diff
@@ -361,7 +361,15 @@ to find your schedule for a specific day of the week.
...
+
+* Locate to the folder to save `ugteach.jar` and go to the folder `data`.
```
Comment (teamotitan) [inline docs/UserGuide.md:376]: If you wanna keep, I want to suggest that:

- To locate the data file, go to the folder `data`, and find a file called `ugteach.json`.
- You can then duplicate the data file and save it somewhere else.

**Review by teamotitan — COMMENTED** [review]
LGTM, some suggestions

**Review by Feng1231 — APPROVED** [review]
lgtm

**Pull request merged**